Jenkinsfile

```diff
@@ -13,46 +13,42 @@ pipeline {
   }
 
   stages {
+
+    stage('Checkout Submodules'){
+      steps {
+        sh 'git submodule update --init --recursive'
+      }
+    }
+
+    stage("Build Docker Image"){
+        steps { sh './build.sh' }
+    }
+
```

Comment:
We chose to put changelog validation first, so that if something is wrong with it the pipeline fails fast (and the check is very quick)

Reply:
Validate uses the image produced by the build stage